embedly/embedly/api/views.py

```diff
@@ -34,8 +34,8 @@ def version():
     )
 
 
-@blueprint.route('/extract')
-def extract_urls():
+@blueprint.route('/extract', methods=['GET'])
+def extract_urls_v1():
```

Comment:
Do we even wait to still support submitting requests via GET? (and risk having them appear in logs or anywhere else -- versus just supporting POST and failing on any other method)